rfc/0003-indexmanagement.md

```diff
@@ -0,0 +1,156 @@
+# Meta
+
+ - RFC Name: Index Management
+ - RFC ID: 0003-indexmanagement
+ - Start Date: 2015-12-14
+ - Owner: Simon Baslé (@simonbasle)
+ - Current Status: Draft
+
+# Summary
+The aim of the "`Index Management`" RFC is to offer a simplified API in the `BucketManager` to list, create and drop indexes, for the majority of use cases. In this sense, it is focused on GSI indexes and assumes a few other givens. For cases that don't fit in this picture, the fallback for the user is to craft appropriate `N1QL` queries (eg. in Java using the `Index` fluent API).
+
+# Motivation
+The goal is to abstract away regularly needed queries relative to indexes and provide a simple API. This way the user can focus on his business related `N1QL` queries, without having to think twice about the boilerplate of creating indexes.
```

Comment:
wording: "This way the user can focus on N1QL queries without having to thing twice..."